contracts/standard/permission/ExposedArbitrableBlacklist.sol

```diff
@@ -0,0 +1,16 @@
+// ONLY FOR TESTING PURPOSES
```

Comment:
Why would we need this contract for testing? ArbitrableBlacklist and CentralizedArbitrator should be enough.

Reply:
For `executeRuling`, we can test it indirectly with `rule`. Adding an extra contract would complicate the codebase.

Reply:
See: https://github.com/kleros/kleros-interaction/blob/master/contracts/standard/arbitration/Arbitrable.sol#L68

Reply:
Thanks for the review.

I tried to use `rule` to test it but there is a problem.

Situation: `rule` function in `Arbitrable` has `onlyArbitrator` modifier, so only the arbitrator can succesfully execute it. And the arbitrator in this case is a `CentralizerArbitrator` instance. (`ArbitrableBlacklist` constructor parameter 1) 

What I'm trying:
`await arbitrableBlacklist.rule(disputeID, RULING.BLACKLIST, {from: centralizedArbitrator.address})`

What I get: "Error: sender account not recognized".

As far as I understand Ganache complains because I don't have the private key for that address.

Googled it for 2 hours but couldn't find a solution, feels like I'm trying to do something impossible.

How should I proceed?

Reply:
@ferittuncer 
I think you can use this:

https://github.com/kleros/kleros-interaction/blob/bdf08594564fde4fcdfa22bb04bed33809798807/contracts/standard/arbitration/CentralizedArbitrator.sol#L87

Right @clesaege ?

Reply:
@ferittuncer 
I think you can use this:
https://github.com/kleros/kleros-interaction/blob/bdf08594564fde4fcdfa22bb04bed33809798807/contracts/standard/arbitration/CentralizedArbitrator.sol#L87

Right @clesaege ?

Reply:
Thank you @epiqueras it seems using `giveRuling` is doing the job.
Using it however broken my balance assertions, then I realized I have to take account transaction costs in my assertions. I'll update those and send a commit.

Also I realized that transaction cost is always `gasUsed * 10^11`. Weird. It doesn't care what gasPrice I set from Ganache, and pads the value with eleven zeros. Looks like this is a bug of Truffle suite. I'll triple check and report it.